practice/src/javascripts/products/productList.js

```diff
@@ -0,0 +1,14 @@
+import { fetchProducts } from '../services/productService';
+import { ProductCard } from '../templates/product';
+
+// Function to render products on the page
+async function productList() {
```

Comment:
I suggest using `renderProductList` instead of `productList`

Reply:
Function names should use verbs as prefixes.